### ISSUE TITLE
Server shutting down when using CASE keyword

### ISSUE THREAD

**Issue by JacobDer** [nanotee/sqls.nvim#7]
When I attempt to use CASE in a query, the server shuts down right after I type the word. 

Expected behavior: After typing ```SELECT CASE W```, there should be suggestions for keywords which begin with 'W'. 

Actual behavior: No more suggestions are made and server turns off. If I press esc to return to normal mode in Neovim, I also get a message ```Client 1 quit with exit code 2 and signal 0```.

My apologies if this isn't even an issue with sqls.nvim, and thanks in advance nanotee!

**Comment by nanotee** [comment]
Very odd bug... I can reproduce it in VSCode and in Neovim without loading `sqls.nvim` so it's likely a problem with the server

**Comment by JacobDer** [comment]
I see. Perhaps I should open this in lighttiger2505's sqls repo then?

**Comment by nanotee** [comment]
Yes, unfortunately there's not much I can do since I don't maintain the server
